lib/phoenix_test/playwright/frame.ex

```diff
@@ -24,6 +24,30 @@ defmodule PhoenixTest.Playwright.Frame do
     |> unwrap_response(& &1.result.value)
   end
 
+  def screenshot(frame_id, path, opts \\ []) do
+    params =
+      opts
+      |> Keyword.validate!(
+        # Playwright options: https://playwright.dev/docs/api/class-page#page-screenshot
+        full_page: true,
+        omit_background: false
```

Comment:
I tried adding support for other keys (like `:quality` on JPGs), but Playwright seemed to either ignore them or just fail to produce a screenshot entirely. I figure we can revisit in the future to add support for more of Playwright's options.